ui/lib/apps/Statement/utils/tableColumns.tsx

```diff
@@ -213,144 +97,350 @@ Warnings: ${getValueFormat('short')(rec.sum_warnings, 0, 1)}`
+): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_process_time',
+    'max_process_time',
+    'total_process_time',
+    'ns',
+    rows
+  )
+}
+
+function avgTotalWaitColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_wait_time',
+    'max_wait_time',
+    'total_wait_time',
+    'ns',
+    rows
+  )
+}
+
+function avgBackoffColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_backoff_time',
+    'max_backoff_time',
+    'backoff_time',
+    'ns',
+    rows
+  )
+}
+
+function avgWriteKeysColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_write_keys',
+    'max_write_keys',
+    'avg_write_keys',
+    'short',
+    rows
+  )
+}
+
+function avgProcessedKeysColumn(
+  tcf: TableColumnFactory,
+  rows?: any[]
+): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_processed_keys',
+    'max_processed_keys',
+    'avg_processed_keys',
+    'short',
+    rows
+  )
+}
+
+function avgTotalKeysColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_total_keys',
+    'max_total_keys',
+    'avg_total_keys',
+    'short',
+    rows
+  )
+}
+
+function avgPreWriteColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_prewrite_time',
+    'max_prewrite_time',
+    'prewrite_time',
+    'ns',
+    rows
+  )
+}
+
+function avgCommitColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_commit_time',
+    'max_commit_time',
+    'commit_time',
+    'ns',
+    rows
+  )
+}
+
+function avgGetCommitTsColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_get_commit_ts_time',
+    'max_get_commit_ts_time',
+    'get_commit_ts_time',
+    'ns',
+    rows
+  )
+}
+
+function avgCommitBackoffColumn(
+  tcf: TableColumnFactory,
+  rows?: any[]
+): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_commit_backoff_time',
+    'max_commit_backoff_time',
+    'commit_backoff_time',
+    'ns',
+    rows
+  )
+}
+
+function avgResolveLockColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_resolve_lock_time',
+    'max_resolve_lock_time',
+    'resolve_lock_time',
+    'ns',
+    rows
+  )
+}
+
+function avgLocalLatchWaitColumn(
+  tcf: TableColumnFactory,
+  rows?: any[]
+): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_local_latch_wait_time',
+    'max_local_latch_wait_time',
+    'local_latch_wait_time',
+    'ns',
+    rows
+  )
+}
+
+function avgWriteSizeColumn(tcf: TableColumnFactory, rows?: any[]): IColumn {
+  return avgMaxColumn(
+    tcf,
+    'avg_write_size',
+    'max_write_size',
+    'avg_write_size',
+    'bytes',
+    rows
+  )
+}
+
+function avgPreWriteRegionsColumn(
+  tcf: TableColumnFactory,
+  rows?: any[]
+): IColumn {
+  return avgMaxColumn(
```

Comment:
I prefer not to provide `avgMaxColumn`. This pattern will not last long.. Maybe total LOC will be even smaller if it is removed.

Reply:
How about keeping the avgMaxColumn but remove all other methods which call the avgMaxColumn?